spec/factories/bikes.rb

```diff
@@ -14,6 +14,14 @@
     cycle_type { CycleType.slugs.first }
     propulsion_type { "foot-pedal" }
 
+    trait :with_image do
```

Comment:
Does this mean that every bike factory now adds an image by default? If so, why do that rather than having it be a separate factory? Does this have a negative impact on test speed?

Reply:
> Does this mean that every bike factory now adds an image by default?

Not by default, only if we pass the trait to the factory -- so `Factory.create(:bike, :with_image)`  will create an associated image, but `Factory.create(:bike)` won't.

Reply:
Sweet addition. Cool new thing!